Hour/Hour/Model/ActivityModel.swift

```diff
@@ -14,6 +14,7 @@ public struct ActivityModel {
     var UID: UUID
     var desc: String?
     var bgColor: UIColor
+    var emojiTitle: String = ""
```

Comment:
Please roll this out for this commit, this is defined in task H-2

Reply:
Done